index.html

```diff
@@ -74,44 +86,239 @@
+
             <div class="service__info">
                 <p class="service__info-text">The Wedding Shop is a multi-award-winning gift list service that has been supporting couples for three decades. It’s often a whirlwind of wedmin and emotions, but we’re here to help, and thrilled to be a part of your exciting journey.</p>
                 <button class="button service__info-btn">Our Service Explained</button>
             </div>
+
         </section>
 
         <section class="floating">
+
             <div class="floating__block">
-               <h2 class="handwritten-title">30 Years Experience</h2>
+
+               <h2 class=" floating__block-handwritten handwritten-title">30 Years Experience</h2>
                <img class="floating__img" src="img/float.jpg"/>
+
             </div>
             
             <div class="textbox">
+
                 <p class="textbox__brand-name">The Wedding Shop</p>
                 <h2 class="heading">Our Promise</h2>
                 <p class="textbox__info">We want your experience to be memorable, meaningful and a lot of fun. Most of all, we want to help you choose gifts you’ll treasure for years to come.</p>
                 <button class="button textbox__info-btn">Our Service</button>
+
             </div>
+
         </section>
         
+        <section class="articles">
+
+            <div class="separator"></div>
+
+            <h2 class="articles__header heading">We're Dedicated To Putting You At The Heart Of <span class="coral">Everything</span> We Do.</h2>
+
+        </section>
+
+        <section class="gallery">
+
+            <ul class="gallery__tile">
+
+                <div class="gallery__tile-row">
+                    <li><img class="gallery__tile-item"src="img/tile1.jpg"/></li>
+                    <li><img class="gallery__tile-item" src="img/tile2.jpg"/></li>
+                </div>
+
+                <div class="gallery__tile-row">
+                    <li><img class="gallery__tile-item" src="img/tile3.jpg"/></li>
+                    <li><img class="gallery__tile-item" src="img/tile4.jpg"/></li>
+                </div>
+
+            </ul>
+
+            <ul class="gallery__tile">
+
+                <div class="gallery__tile-row">
+                    <li><img class="gallery__tile-item" src="img/tile5.jpg"/></li>
+                    <li><img class="gallery__tile-item" src="img/tile6.jpg"/></li>
+                </div>
+
+                <div class="gallery__tile-row">
+                    <li><img class="gallery__tile-item-hide" src="img/tile7.jpg"/></li>
+                    <li><img class="gallery__tile-item-hide" src="img/tile8.jpg"/></li>
+                </div>
+
+            </ul>
+
+        </section>
+
+        <section class="create-list">
+
+            <img class="logo__bow bow-large" src="img/Bow.png">
+            <h2 class="create-list__header heading">Start Your Wedding Gift List</h2>
+            <div class="create-list-btn button">Create a List</div>

```

Comment:
I think you meant .create-list__btn (where .create-list is a block and .create-list__btn is its element)?

Reply:
Yes I did, have changed!